File: Data Structures & Algorithms/longest-consecutive-sequence/submission-0.py
class Solution:
    def longestConsecutive(self, nums: List[int]) -> int:
        myset = set(nums)
        longest = 0
        for n in nums:
            total = 0
            if (n-1) not in myset:
                while (n+total) in myset:
                    total += 1
                longest = max(total, longest)
        return longest

File: Data Structures & Algorithms/find-minimum-in-rotated-sorted-array/submission-0.cpp
class Solution {
public:
    int findMin(vector<int> &nums) {
        int l = 0, r = nums.size()-1;
        if(nums[l]>nums[r]){
            while(l != (r-1)){
                int m = (l+r)/2;
                if(nums[m]>nums[r]){
                    l = m;
                }
                else{
                    r = m;
                }
            }
            cout << nums[l] << l << endl << nums[r] << r <<endl;
            return min(nums[l], nums[r]);
        }
        else return nums[l];
    }
};
